src/systems/sound-effects-system.js

```diff
@@ -203,7 +217,24 @@ export class SoundEffectsSystem {
     );
   }
 
+  stopAllPositionalAudios() {
```

Comment:
Does this get used?

Reply:
Yes, when the pref flips on: https://github.com/mozilla/hubs/pull/2443/files#diff-42eb0478cd8f412d222db2e8229724daR115

Reply:
Oh wow sorry, somehow when I did a search just now it didnt find anything, bizarre